director/projects/api/views/snapshots.py

```diff
@@ -137,7 +137,7 @@ def create(self, request: Request, pk: int) -> Response:
             raise PermissionDenied
 
         # Create the snapshotter and event for the snapshot
-        snapshotter = ProjectSnapshotter(settings.STENCILA_PROJECT_STORAGE_DIRECTORY)
+        snapshotter = ProjectSnapshotter(settings.STORAGE_DIR)
```

Comment:
I would prefer having a `STENCILA` prefix on our settings because a lot of third party libraries are bad and don't prefix/"namespace" their settings. `STORAGE_DIR` seems like a somewhat common one that could conflict in the future.